### PR TITLE
UserServiceTests_CreateUnitTestsForSignUp

### PR DESCRIPTION
**Pull request by Maksim-Trolina**

### CODE REVIEW

File: Tests/UserServiceTests/Commands/CreateUserCommandTests.cs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            clientMock = new Mock<IRequestClient<InternalCreateUserRequest>>();
+            clientMock
+                .Setup(x => x.GetResponse<OperationResult>(It.IsAny<InternalCreateUserRequest>(), default, default))
+                .Returns(Task.FromResult(responseMock.Object));
+
+            command = new CreateUserCommand(clientMock.Object, validatorMock.Object);
+        }
+
+        [Test]
+        public void CreateUserCommand_ValidationException_IncorrectUserData()
```
Comment (Cerberous007) [inline Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: Names of methods should be in CamelCase

File: Tests/UserServiceTests/Commands/CreateUserCommandTests.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+using MassTransit;
+using DTO.BrokerRequests;
+using DTO;
+using System.Threading.Tasks;
+using FluentValidation.Results;
+using Kernel.CustomExceptions;
+
+namespace UserServiceTests
+{
+   public class CreateUserCommandTests
```
Comment (nmamro) [inline Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: Move it to the right a bit

File: Tests/UserServiceTests/Commands/CreateUserCommandTests.cs (hunk 1 above)
Comment (Maksim-Trolina) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: > Names of methods should be in CamelCase

https://docs.microsoft.com/ru-ru/dotnet/core/testing/unit-testing-best-practices

So even at Microsoft they call tests

Comment (nmamro) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: > > Names of methods should be in CamelCase
> 
> https://docs.microsoft.com/ru-ru/dotnet/core/testing/unit-testing-best-practices
> 
> So even at Microsoft they call tests

Where do they say about underlines?

File: Tests/UserServiceTests/Commands/CreateUserCommandTests.cs (hunk 2 above)
Comment (Maksim-Trolina) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: > Move it to the right a bit


These three lines?

**Review by denis-kriv — APPROVED** [review]

File: Tests/UserServiceTests/Commands/CreateUserCommandTests.cs (hunk 2 above)
Comment (nmamro) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: > > Move it to the right a bit
> 
> These three lines?

`public class CreateUserCommandTests` should be moved a bit to the right, does not code like

```
namespace N
{
     class X
         {
         }
}
```
bother you?

File: Tests/UserServiceTests/Commands/CreateUserCommandTests.cs (hunk 1 above)
Comment (Maksim-Trolina) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: > > > Names of methods should be in CamelCase
> > 
> > 
> > https://docs.microsoft.com/ru-ru/dotnet/core/testing/unit-testing-best-practices
> > So even at Microsoft they call tests
> 
> Where do they say about underlines?


They call all tests so

Comment (nmamro) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: > > > > Names of methods should be in CamelCase
> > > 
> > > 
> > > https://docs.microsoft.com/ru-ru/dotnet/core/testing/unit-testing-best-practices
> > > So even at Microsoft they call tests
> > 
> > 
> > Where do they say about underlines?
> 
> They call all tests so

And where in the recommendations they say "call tests using underlines?"

Comment (Maksim-Trolina) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: > And where in the recommendations they say "call tests using underlines?"


They don’t write like that. But they do it

File: Tests/UserServiceTests/Commands/CreateUserCommandTests.cs (hunk 2 above)
Comment (Maksim-Trolina) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: > > > Move it to the right a bit
> > 
> > 
> > These three lines?
> 
> `public class CreateUserCommandTests` should be moved a bit to the right, does not code like
> 
> ```
> namespace N
> {
>      class X
>          {
>          }
> }
> ```
> 
> bother you?


Fixed

File: Tests/UserServiceTests/Commands/CreateUserCommandTests.cs (hunk 1 above)
Comment (Maksim-Trolina) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: > Names of methods should be in CamelCase

Fixed

Comment (Maksim-Trolina) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: > > > > > Names of methods should be in CamelCase
> > > > 
> > > > 
> > > > https://docs.microsoft.com/ru-ru/dotnet/core/testing/unit-testing-best-practices
> > > > So even at Microsoft they call tests
> > > 
> > > 
> > > Where do they say about underlines?
> > 
> > 
> > They call all tests so
> 
> And where in the recommendations they say "call tests using underlines?"

Fixed

**Review by nmamro — APPROVED** [review]

**Comment by nmamro** [comment]
Funny that we have Initialize(), Initialization() and even mb Init() in different test projects in one solution =)

File: Tests/UserServiceTests/Commands/CreateUserCommandTests.cs (hunk 1 above)
Comment (Cerberous007) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: But only you use underscores in names of test methods. I think, it`s not good practise

Comment (Maksim-Trolina) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: > But only you use underscores in names of test methods. I think, it`s not good practise


I have corrected

Comment (Cerberous007) [reply Tests/UserServiceTests/Commands/CreateUserCommandTests.cs]: Sorry, I didn`t see it

**Review by Cerberous007 — APPROVED** [review]

**Review by DanilFadeeev — APPROVED** [review]

**Review by LK777LK — APPROVED** [review]

**Pull request merged**